Assets/SteamVR_Unity_Toolkit/Scripts/Controls/3D/VRTK_Control.cs

```diff
@@ -42,7 +52,16 @@ public float getValue()
             return value;
         }
 
-        public virtual void Start()
+        /// <summary>
+        /// Returns the current value mapped onto a range between 0% and 100%.
+        /// </summary>
+        /// <returns>Normalized Value</returns>
+        public float getNormalizedValue()
```

Comment:
Actually i dont think it will, we don't have docs for the VRTK_Control do we? I'm guessing you never use this class, so it doesn't need documenting?